src/interface/src/app/treatments/treatment-layout/treatment-layout.component.ts

```diff
@@ -10,6 +11,10 @@ import { TreatmentMapComponent } from '../treatment-map/treatment-map.component'
   styleUrl: './treatment-layout.component.scss',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class TreatmentLayoutComponent {
-  constructor() {}
+export class TreatmentLayoutComponent implements OnDestroy {
+  constructor(private treatmentsState: TreatmentsState) {}
+
+  ngOnDestroy(): void {
+    this.treatmentsState.reset();
```

Comment:
We shouldn't be doing this - Im addressing this issue on https://github.com/OurPlanscape/Planscape/pull/1746